src/directives/automation/editor/dpEditorListsGrid.js

```diff
@@ -96,59 +93,62 @@
         return row.IdSubscribersList === selectedItemId;
       };
 
-      $scope.$on('COMPONENT_LIST.CONFIRM_SELECTION', function() {
+      $scope.confirmListSelection = function() {
         if (!$scope.gridModel.selectedItem) {
           return;
         }
-        var data;
-        var newComponentData;
-        var oldComponentData;
-        var path;
         var selectedComponent = selectedElementsService.getSelectedComponent();
-
-        if (selectedComponent.type === COMPONENT_TYPE.ACTION) {
-          path = 'operation.suscriptionList';
-          data = {
-            operation: {
+        // TODO: Analize else branch
```

Comment:
Is this TODO still pending?

Reply:
yes that is right, the goal is resolve the visual issue in this PR, and not improve the logic. I could detect the else branch is not used, with a lazy test

Reply:
✔️